nike_bot.py

```diff
@@ -1,63 +1,63 @@
-if __name__ == "__main__":
-	import requests
-	import time
-	import twitter
+#if __name__ == "__main__":

```

Comment:
entrypoint for a python app, it runs the app through terminal command `python nike_bot.py` without any other arguments